src/aditi/FormFieldValidations.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * 1) Open the form in browser and verify Title is “TECHNOCREDITS”. 
+ * 2) Fill all details in the form, for Gender radio button if ‘Female’ radio button 
+ * is by default selected then select ‘Male’ radio button and vice versa.
+ * 3) Reset the form, for that click on “Reset form in same tab” button.
+ * 4) Verify Reset works properly on your text field, radio button and checkbox. (For each type, min one validation is required.)
+ * 5) Click on the “Go and Practice for it” Button.
+ * 6) Website redirect to “automationbykrishna.com”.
+ * 7) Verify title and URL of redirected page website.
+ * Title should be “Login Signup Demo” and the URL should be “http://automationbykrishna.com/”.
+ */
+package aditi;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.InvalidArgumentException;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class FormFieldValidations {
+	WebDriver driver = new ChromeDriver();
+
+	void setElementValueOnForm() {
+		try {
+			driver.get("file:///Users/apple/Documents/Class_Sessions/PracticeWebPages/SeleniumAssignment_1.html");
+		} catch (InvalidArgumentException invalidArgumentException) {
+			System.out.println("Please enter Correct filepath");
+		}
+
+		// Verifying Title of the web page in browser window
+		if (driver.getTitle().equals("TECHNOCREDITS"))
+			System.out.println("Test Case Pass - Correct landing page - Title matches to -> " + driver.getTitle());
+		else
+			System.out.println("Test case fail - landing page URL is: " + driver.getCurrentUrl());
+
+		// Set value of web elements in form
+		try {
+			driver.findElement(By.id("first name")).sendKeys("Aditi");
+			driver.findElement(By.id("last name")).sendKeys("Gajipara");
+			driver.findElement(By.id("E-mail")).sendKeys("aditi@gmail.com");
+			driver.findElement(By.id("Company Name")).sendKeys("Chase");
+			driver.findElement(By.id("continents")).sendKeys("North America");
+			driver.findElement(By.id("entry2")).click();
+			driver.findElement(By.id("knownlanguages")).sendKeys("JAVA");
+			driver.findElement(By.id("python")).click();
+			driver.findElement(By.id("javascript")).click();
+			driver.findElement(By.id("google")).click();
+			driver.findElement(By.id("termsAndConditions")).click();
+
+			// Checking validation for radio button condition
+			if (driver.findElement(By.id("femaleG")).isSelected()) {
+				driver.findElement(By.id("maleG")).click();
+				System.out.println("Default Selection was Female - Radio Button selection changed to Male");
+			} else {
+				driver.findElement(By.id("femaleG")).click();
+				System.out.println("Default Selection was Male - Radio Button selection changed to Female");
+			}
+		} catch (NoSuchElementException elementException) {
```

Comment:
This is the wrong dear, We should never handle NoSuchElementException when we performing an action on it. If any of the above steps failed then your test case will not return from which step your case failed.